### PR TITLE
First multiprocessing version of tiler

### PR DESCRIPTION
**Pull request by gjn**
This is first working version of multiprocessing tiler. To enable it, you need to adapt the tms.cfg.

This can still be improved; especially the session handling. So I'll continue to work on that.

### CODE REVIEW

**Comment by gjn** [comment]
@loicgasser I've adapted a bit, removed most of the logging (because logging drags down performance heavily; I think we should only log failure) and was able to process 10'000 tiles in under 3 minutes:
`2015-07-09 10:12:28,788 | forge.lib.tiler                  | INFO     | [30888] It took 0:02:57.687965 to create 10000 tiles on S3.`

Thats 55 tiles per second.

**Comment by gjn** [comment]
I'm doing a full re-processing now to see how long it will take.

**Comment by gjn** [comment]
Won't do full processing, but with 55 tiles per second, we can generate whole pyramid with dm25 data in 1 day and 9 hours. Not too shabby. (6.5 million tiles)

I'll let it run for around an hour to see if performance remains stable.

**Comment by loicgasser** [comment]
is that still WIP. Seems pretty advanced to me.

**Comment by gjn** [comment]
Ok, I'll remove the WIP.

Note however, that I found some strange behaviour (some processes going stale...but not often. This needs to be adressed). Anyway, we can always disable multiprocessing in the configuration.

Please merge if you like it.

File: forge/lib/tiler.py [hunk 1]
```diff
@@ -51,66 +149,56 @@ def __init__(self, configFile):
...
-                ti = t1 - self.t0
-                self.logger.info('It took %s HH:MM:SS to write %s tiles' % (str(datetime.timedelta(seconds=ti)), count))
-                count += 1
-            else:
-                # One should write an empyt tile
-                self.logger.info('Skipping %s because no features have been found for this tile' % bucketKey)
+            job = (self.config, self.tileMinZ, self.tileMaxZ, bounds, tileXYZ, self.t0, bucket)
+            jobs.append(job)
+
+        print len(jobs)
```
Comment (loicgasser) [inline forge/lib/tiler.py]: Did you keep this pring on purpose?

**Pull request merged**